setup.py

```diff
@@ -61,6 +61,7 @@
         'python-coveralls',
         'requre',
         'pre-commit',
+        'types-click',
```

Comment:
The `types-click` [documentation](https://pypi.org/project/types-click/) says:

> Note: The click package includes type annotations or type stubs since version 8.0. Please uninstall the types-click package if you use this or a newer version.

Do we need to add this extra dependency if we're developing with fresh `click`?

Reply:
Thanks, looks like only mention in mypy.ini was needed